### ISSUE TITLE
Templates don't appear to be working for me...

### ISSUE THREAD

**Issue by PhysicsLagomorph** [Grapheel/IRIS#9]
### About me and my system

1. I'm using IRIS as a [volunteer]
2. I am using assistive technology: [No]
3. My device is a: A) Samsung mobile phone, B) Microsoft surface
4. My operating system is: A) Android B) Windows 10
5. The web browser I'm using is: A) Android mobile browser B) Firefox
6. The version of the web browser is: 

### Bugs found with description

I found the following bug(s):

When I click an image it has a text entry box but selecting different templates does not appear to change anything. I have check this on both my phone and my laptop. I can enter a description fine in the text entry box but there does not appear to be an inbuilt templates.

**Comment by tingard** [comment]
You are right that the template selector is currently broken, sorry for the confusion!

Reworking the templates is currently under-way, I'll post here when the new versions are released! (Probably next week sometime)
